repository-data/development/src/main/resources/hcm-actions.yaml

```diff
@@ -6,3 +6,7 @@ action-lists:
       /content/documents/lks/landing-pages: reload
   - 1.0.2:
       /content/documents/lks/minihubs: reload
+  - 1.0.3:
+      /content/documents/lks/guidance-pages: reload
+      /content/documents/lks/minihubs: reload
+      /content/documents/administration/labels/global: reload
```

Comment:
@thojava - `/content/documents/administration/labels/global` didn't reload on my local instance. Do you think we would need to move this onto `repository-data/application/src/main/resources/hcm-actions.yaml` ?

Reply:
Done